configs/restorers/iconvsr/iconvsr_vimeo90k_bd.py

```diff
@@ -16,7 +16,7 @@
     pixel_loss=dict(type='CharbonnierLoss', loss_weight=1.0, reduction='mean'))
 # model training and testing settings
 train_cfg = dict(fix_iter=5000)
-test_cfg = dict(metrics=['PSNR', 'SSIM'], crop_border=0, convert_to='y')
```

Comment:
Is removing 'SSIM' intended?

Reply:
Yes, as it would slow down the training. Therefore I put only PSNR here.

Reply:
Users may remove 'SSIM', or together with 'PSNR' themselves if they want to speed up training. It's better to leave it here since our result page contains both SSIM and PSNR.